src/index.js

```diff
@@ -7,4 +7,187 @@ import './css/base.scss';
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
 
+import ApiFetch from './ApiFetch';
+import domUpdates from './domUpdates';
+import Traveler from './traveler';
+
 console.log('This is the JavaScript entry file - your code begins here.');
+
+window.addEventListener('load', onLoad);
+
+
+function onLoad() {
+  const loginBtn = document.querySelector('.login-button')
+  loginBtn.addEventListener('click', enterLogin)
+
+  // displayPastTrips()
+
+}
+
+function enterLogin() {
+  const username = document.getElementById('username').value
+  const password = document.getElementById('password').value
+  const loginValue = parseInt(username.match(/\d+/gi).pop())
+  console.log("VALUE", loginValue)
+  const removeLogin = document.querySelector('.entry-form')
+  if (username.includes('traveler') && (username.split('traveler')[1] < 51 && username.split('traveler')[1] > 0) && password === 'travel2020') {
+    console.log(username + ' is logged in!!!')
+    removeLogin.classList.add('hidden')
+    getData(loginValue)
+  } else {
+    alert("WRONG PASSWORD")
+  }
+
+}
+
+//get the value of username
+
+// var str = "traveler50";
+// var res = str.match(/\d+/gi);
+
+
+function getData(loginValue) {
+  const api = new ApiFetch();
+  const tripData = api.getTripsData()
+  const travelerData = api.getTravelersData()
+  console.log("TD", travelerData)
+  const destinationData = api.getDestinationsData();
+  function _createTraveler(data) {
+    createTraveler(data, loginValue)
```

Comment:
A nice use of closure here to keep your function private and protect your created traveler.